cuppa-surefire/src/main/java/org/forgerock/cuppa/maven/surefire/CuppaSurefireProvider.java

```diff
@@ -26,6 +32,39 @@
      */
     public CuppaSurefireProvider(ProviderParameters parameters) {
         this.providerParameters = parameters;
+        Properties properties = parameters.getProviderProperties();
+        tags = new Tags(getTags(properties), getAntiTags(properties));
+    }
+
+    private Set<String> getTags(Properties properties) {
+        String groups = properties.getProperty("groups");
```

Comment:
Yes it is but as surefire as implicit support for TestNG and JUnit it will pick up the `-Dgroups=` from the command line and it will override the value set in `properties.getProperty("groups");`. So its handled for us.